packages/api/test/scripts/worker-globals.js

```diff
@@ -1,6 +1,11 @@
 export const caches = {
   default: {
-    match: () =>  null,
+    match: () => null,
     put: () => {}
   }
 }
+
+export const SALT = 'test-salt'
```

Comment:
Will we do the ENV setup in follow up?

Reply:
no? not sure I understand?

Reply:
If we will merge this and do the environment set up with wrangler secrets as a follow up PR

Reply:
This simulates having the env passed in as magic globals

Reply:
yes, my point was for production, but @alanshaw already pointed me to the README with this. I didn't realize we already had it